pastas/stressmodels.py

```diff
@@ -892,6 +892,54 @@ def to_dict(self, series=True):
         }
         return data
 
+    def variance_gain(self, model, istress=None):
```

Comment:
Good idea to add r as kwarg here, allows calculating uncertainty as function of distance. Note to self to open issue later, not requisite for this PR.